frontend/micro-ui/web/micro-ui-internals/package.json

```diff
@@ -14,7 +14,10 @@
     "packages/modules/pgr",
     "packages/modules/pt",
     "packages/modules/tl",
-    "packages/modules/ws"
+    "packages/modules/ws",
+    "packages/modules/commonPt",
+    "packages/modules/templates",
```

Comment:
Template is not required
The module requiring changes should be there in workspace